.github/workflows/deploy_beta_testing.yml

```diff
@@ -18,6 +18,10 @@ jobs:
           distribution: 'zulu'
           java-version: '11'
 
+      - name: Enable API Session Auth feature flag
+        working-directory: src/main/resources/META-INF
+        run: echo -e "dataverse.feature.api-session-auth=true" >> microprofile-config.properties
```

Comment:
Will this append the line over and over again, on every build?

Reply:
@pdurbin No. This change only takes effect during the action lifetime and the content of the git repository is not modified.

Reply:
Oh. Duh. Make sense. Thanks! 😄